### PR TITLE
source-piwik contribution from btkcodedev

### PR DESCRIPTION
**Pull request by btkcodedev**
Closes https://github.com/airbytehq/airbyte-internal-issues/issues/9821
Auto-generated PR by the Connector Builder for source-piwik

### CODE REVIEW

File: docs/integrations/sources/piwik.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+| modules | id | DefaultPaginator | ✅ |  ❌  |
+| meta-sites | id | DefaultPaginator | ✅ |  ❌  |
+| meta-sites_apps | id | DefaultPaginator | ✅ |  ❌  |
+| meta-sites_apps-with-meta-sites | id | DefaultPaginator | ✅ |  ❌  |
+| container-settings_settings |  | DefaultPaginator | ✅ |  ❌  |
+| apps | id | DefaultPaginator | ✅ |  ❌  |
+| audit-log_entries | id | DefaultPaginator | ✅ |  ❌  |
+| apps_details | id | DefaultPaginator | ✅ |  ❌  |
+| access-control_actions | id | DefaultPaginator | ✅ |  ❌  |
+| access-control_meta-site_permission_user |  | DefaultPaginator | ✅ |  ❌  |
```
Comment (natikgadzhi) [inline docs/integrations/sources/piwik.md:35]: Why doesn't this stream have a primary key defined?

**Comment by addison-grant** [comment]
I was wondering, would it be appropriate to extend this source to enable downloading events and sessions?
API details at https://developers.piwik.pro/en/latest/custom_reports/http_api/http_api.html#tag/Raw-data

I would consider coding it up if so. I didn't see an existing GitHub issue in this repository.

**Pull request merged**